add what is dashboard ent (#900)
vesoft-inc/nebula-docs: docs-2.0/nebula-dashboard-en/1.what-is-dashboard-en.md

# What is Nebula Dashboard Enterprise Edition

Nebula Dashboard (Dashboard for short) is a visualization tool that monitors and manages the status of machines and services in Nebula Graph clusters. This topic introduces Dashboard Enterprise Edition. For details of Dashboard Community Edition, refer to [What is Nebula Dashboard Community Edition](../nebula-dashboard/1.what-is-dashboard.md).

## Features

> - Create a Nebula Graph cluster of specified version, support import nodes in batch, add services with one click, and so on.

Reviewer: delete [, and so on.]

> - Support importing clusters, balancing data, scaling out or in, and so on through visualization interface.

Reviewer: to uniform the style with all sentences, it's better to start the sentence with support, or delete support.

Reviewer: `and so on` is not used often in technical writing. you can use etc or just delete it.

> - Support manage clusters, and audit operation log of clusters within the last 14 days.

Reviewer: Support doing/Support noun

it will be better to omit support , start a sentence with a verb that what users can do.

- Start, stop and restart services through visualization interface.

- Update the configuration of Storage services and Graph services in clusters quickly.

> - Monitor the information of all the services in clusters, including the IP address, version, and monitoring metrics (such as the number of queries, the latency of queries, the latency of heartbeats).

Reviewer 2: ```suggestion
- Monitor the information of all the services in clusters, including the IP address, version, and monitoring metrics (such as the number of queries, the latency of queries, and the latency of heartbeats).
```

- Monitors the status of all the machines in clusters, including CPU, memory, load, disk, and network.

- Monitors the information of clusters, including the information of services, partitions, configurations, and long-term tasks.

## Scenarios

- You want a visualized operation and maintenance monitoring platform for large-scale clusters.

> - You want to monitor key metrics conveniently and quickly, and present multiple key information of the business to ensure the business operates normally.

Reviewer: ensure that +clause 
like a business can operate xx

> - You want to monitor clusters from multiple dimensions(such as the time, aggregate rules, and metrics).

Reviewer: dimensions (such

> - You want to review it and confirm its occurrence time and unexpected phenomena after a failure occurs.

Reviewer 2: ```suggestion
- You want to review the failure after it occurs, confirm when it happened, and view its associated phenomena.
```

> - after a failure occurs, you want to review it and confirm its occurrence time and unexpected phenomena.

Reviewer: After

## Precautions

- The monitoring data will be updated per 7 seconds by default.

- The monitoring data will be retained for 14 days by default, that is, only the monitoring data within the last 14 days can be queried.

- The version of Nebula Graph must be 2.0.1 or later.

> - It's recommend to use the latest version of Chrome to access Dashboard.

Reviewer 2: ```suggestion
- It is recommend to use the latest version of Chrome to access Dashboard.
```

> - Advised to use the latest version of Chrome to access Dashboard.

Reviewer: use the complete form  
Advised --> it's advised to do  
but we use words suggest or recommend often in technical docs.

> - It's recommend to use the official installation package to create or import clusters.

Reviewer 2: ditto

!!! note

>     The monitoring service is supported by Prometheus. The update frequency and retention intervals can be modified. For details, see [Prometheus](https://prometheus.io/docs/prometheus/latest/configuration/configuration/).

Reviewer: to avoid ambiguity

monitoring service -->monitoring feature
